### DISCUSSION TITLE
Require CAPTCHA and cant log in

### DISCUSSION THREAD

**Discussion by JessJiayi** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#253] [Q&A] [answered]
when I start the enroller, the browser pop up. 
but asking the complete CAPTCHA, then browser close in few seconds.
How can I fix it?

**Comment by fakeid30** [comment, marked as answer]
use the alpha branch. the traditional method of captcha based login is deprecated.
